game_patch/misc/player.cpp

```diff
@@ -331,6 +345,9 @@ void player_do_patch()
     // Fix hud msg never disappearing in spectate mode
     players_do_frame_hook.install();
 
+    // Fix jeep cockpit not rendering for any jeeps entered after the first
+    player_animate_entity_parent_hook.install();
```

Comment:
this can be fixed with just one line which changes `jz` at `004A77C3` into `jmp` instead of reimplementing entire `player_animate_entity_parent`. `AsmWriter` is your friend

Reply:
I originally planned to fix it that way, but decided against because I wanted to remove the hardcoded specific treatment for the Jeep - both because it appears unnecessary (and should therefore be removed IMO as a best practice) and because hardcoding it as the stock game does could potentially be limiting for mods down the road. For those reasons, I rewrote the function instead of using the `AsmWriter` approach.

If you would prefer it be addressed via `AsmWriter` I will adjust this PR accordingly.

Reply:
I'm not sure what you mean here. Changing `jz` to `jmp` will effectively remove special treatment for jeeps (or more like disable so it's never executed). I don't see any difference compared to your solution except it's simpler.